run_scans.py

```diff
@@ -134,22 +146,29 @@ def launchTestsArgs(tool, slot, link, chamber, scanmin, scanmax, nevts,
 
   from qcoptions import parser
 
-  parser.add_option("--series", action="store_true", dest="series",
```

Comment:
It's not removed, I alphabetised the options menu, see line 158:

https://github.com/bdorney/vfatqc-python-scripts/blob/develop/run_scans.py